operations/spire/Enable_TPM_node_attestation.md

```diff
@@ -0,0 +1,243 @@
+time="2025-04-02T20:11:20Z" level=info msg="Get DevID (RSA)"
+time="2025-04-02T20:11:20Z" level=info msg="Certifying TPM-residency of keys"
+time="2025-04-02T20:11:21Z" level=info msg="CSR creation complete! [3]"
+Warning: Permanently added 'ncn-w003' (ED25519) to the list of known hosts.
+ncn-w003 is being joined to spire.
+Warning: Permanently added 'ncn-w003' (ED25519) to the list of known hosts.
+spire-tpm-conf.conf                                                                                                                                                                                                                                                                        100%  726     1.3MB/s   00:00
+Warning: Permanently added 'ncn-w003' (ED25519) to the list of known hosts.
+Warning: Permanently added 'ncn-w004' (ED25519) to the list of known hosts.
+Warning: Permanently added 'ncn-w004' (ED25519) to the list of known hosts.
+{"success":true}
+Warning: Permanently added 'ncn-w004' (ED25519) to the list of known hosts.
+2025/04/02 20:11:26 url: https://api-gw-service-nmn.local/apis/tpm-provisioner
+2025/04/02 20:11:26 xname: x3000c0s10b0n0
+2025/04/02 20:11:26 type: ncn
+2025/04/02 20:11:26 req: &{Method:GET URL:https://api-gw-service-nmn.local/apis/tpm-provisioner/authorize?xname=x3000c0s10b0n0&type=ncn Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[] Body:<nil> GetBody:<nil> ContentLength:0 TransferEncoding:[] Close:false Host:api-gw-service-nmn.local Form:map[] PostForm:map[] MultipartForm:<nil> Trailer:map[] RemoteAddr: RequestURI: TLS:<nil> Cancel:<nil> Response:<nil> ctx:{emptyCtx:{}}}
+time="2025-04-02T20:11:26Z" level=info msg="Reading EK certificate from NV index 01c00002"
+time="2025-04-02T20:11:26Z" level=info msg="Get Endorsement Key (RSA)"
+time="2025-04-02T20:11:27Z" level=info msg="Get Attestation Key (RSA)"
+time="2025-04-02T20:11:32Z" level=info msg="Get DevID (RSA)"
+time="2025-04-02T20:11:33Z" level=info msg="Certifying TPM-residency of keys"
+time="2025-04-02T20:11:33Z" level=info msg="CSR creation complete! [3]"
+Warning: Permanently added 'ncn-w004' (ED25519) to the list of known hosts.
+ncn-w004 is being joined to spire.
+Warning: Permanently added 'ncn-w004' (ED25519) to the list of known hosts.
+spire-tpm-conf.conf                                                                                                                                                                                                                                                                        100%  726     1.7MB/s   00:00
+Warning: Permanently added 'ncn-w004' (ED25519) to the list of known hosts.
+```
+
+### Disable TPM attestation on management nodes
+
+To disable the TPM attestation on management nodes the `spire-agent.conf`
+file must be returned to its original configuration on each node TPM attestation
+is turned on. Then a new join token must be created and the node then needs
+to be joined to spire.
```

Comment:
Lowercase "spire" here, but I think caps in all other places...

Reply:
Thanks will updated.